### ISSUE TITLE
Aspect ratio change not working

### ISSUE THREAD

**Issue by Flame69chaos** [jimgraygit/Onion#419]
I wanted to ask about the aspect ratio settings in the NES and Sega Genesis cores (FCEUmm, Genesis Plus GX). I try to change in Video - Aspect Ratio to different settings - 4:3, 8:7 (PAR). Nothing changes. It only works if you change it in the Scaling menu and set Keep Aspect Ratio to ON. But that's still not the case. Does everyone not change the ratio in the cores or just me?

**Comment by SirPrimalform** [comment]
Yeah, I have also found that the aspect ratio settings don't work, meaning that realistically you only get two options: square pixels or stretch to 4:3. There are many systems where neither of those options are exactly correct.
